express/scripts/gnav.js

```diff
@@ -69,7 +69,12 @@ function loadIMS() {
     locale: getLocale(window.location),
     environment: 'prod',
   };
-  loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
+  if (!['www.stage.adobe.com'].includes(window.location.hostname)) {
```

Comment:
@apganapa Do you mind taking a look and see if an existing function called getHelixEnv() in scripts.js can be used for this fix?

Reply:
Otherwise LGTM!

Reply:
shall I go ahead add ims-env variable and ims-url to getHelixEnv() method? that way I can keep these in one place and call the getHelixEnv() and use those variables? @qiyundai

Reply:
I don't think we need to change the function. If getHelixEnv gives you the env name for your conditions to work, then I'm suggesting that we could use that instead of a url check as it's a more tested/trusted approach.

Reply:
getHelixEnv() returns "stage", we need "stg1" as value. hence we need to add this variable and while adding probably another key-value for the ims URL. As is getHelixEnv() is not giving the required values.

Reply:
I think all we want is a boolean value saying we are on stage or not, in which case we can replace the hostname check with `getHelixEnv !== 'stage`, as it has a more consistent understand of whether a hostname is considered 'stage' or not.

I'm not sure if www.stage.adobe.com would be the only place we want to load the stg1 version of the imslib.min.js. If we only check for hostname including `www.stage.adobe.com`, then for example:

- main--express-website--adobe.hlx.page will load prod script while technically being stage.
- localhost will load prod script while technically being stage.

We've always been loading prod script only so many the ask was just to load a different script for only www.stage.adobe.com. If that were the case we don't need to worry about getHelixEnv. Just wanted to provide as much clarity as possible

Reply:
Right now, yes with Akamaized domain stage.adobe.com only we need to load stg1 and ims specific to stage so that it gets back to stage.adobe.com
All the other places like ".page", ".live",  "localhost" across other projects there is no condition and they are loading Prod IMS script so we will continue to do that to have it consistent.